lib/ops/resources/gravity/gravity.go

```diff
@@ -224,7 +228,7 @@ func (r *Resources) Create(req resources.CreateRequest) error {
 	case "":
 		return trace.BadParameter("missing resource kind")
 	default:
-		return trace.BadParameter("unsupported resource %q, supported are: %v",
+		return trace.NotImplemented("unsupported resource %q, supported are: %v",
```

Comment:
`NotImplemented` is used to differentiate from a `BadParameter` coming from an arbitrary API other than here - this needs to support the resource handling fallback from the enterprise version. There might be a better way of handling this.

Reply:
Reverted to using `trace.BadParameter`.